tensorflow/core/kernels/dml_broadcast_to_op.cc

```diff
@@ -113,20 +110,6 @@ class DmlBroadcastToKernel : public DmlKernel {
                                  &identity_desc};
     Initialize(ctx, std::move(tensors), op_desc);
   }
-
-  StatusOr<DmlGpuEvent> Compute(DmlKernelContext* ctx) const override {
-    // Currently, 64-bit integers in DML are emulated using 32-bit integers
-    // using striding to emulate a larger type. Because we can't guarantee that
-    // our output tensor's memory is zero'd, we need to do so manually prior to
-    // running running gather.
-    Tensor* output = ctx->GetOutputTensor(0);
-
-    if (Is64BitIntegerType(output->dtype())) {
-      ctx->ZeroBuffer(ctx->CreateBufferForTensor(*output));
-    }
-
-    return DmlKernel::Compute(ctx);
-  }
 };
 
 #define REGISTER_KERNEL(type)                                         \
```

Comment:
I don't think it's necessary to constrain Tidx because the operator only supporrts DT_INT32 and DT_INT64. Probably cleaner to just have a single REGISTER_KERENL_BUILDER macro and remove the type constraint.

Reply:
Sure, this is common to a lot of our operators as well. This PR is already pretty big so I'll keep it this way for now.